kDrive/UI/View/Upsale/UpsaleViewController.swift

```diff
@@ -0,0 +1,229 @@
+/*
+ Infomaniak kDrive - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreUIKit
+import kDriveCore
+import kDriveResources
+import UIKit
+
+public class UpsaleViewController: UIViewController {
+    let titleImageView = UIImageView()
+
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        // Some factorisation can be done in kDrive
+        label.font = UIFont.systemFont(ofSize: UIFontMetrics.default.scaledValue(for: 22), weight: .bold)
+        label.textColor = KDriveResourcesAsset.titleColor.color
+        label.numberOfLines = 1
+        label.textAlignment = .center
+        label.text = KDriveStrings.Localizable.obtainkDriveAdTitle
+        return label
+    }()
+
+    let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = .preferredFont(forTextStyle: .body)
+        label.textColor = KDriveResourcesAsset.primaryTextColor.color
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.text = KDriveStrings.Localizable.obtainkDriveAdDescription
+        return label
+    }()
+
+    let freeTrialButton = IKLargeButton(frame: .zero)
+
+    let dismissButton = IKLargeButton(frame: .zero)
+
+    let scrollView = UIScrollView()
+
+    let containerView = UIView()
+
+    let bulletPointsView = UIView()
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = KDriveResourcesAsset.backgroundColor.color
```

Comment:
Wrong color is used compared to the figma

Reply:
Updated the screenshots

Reply:
Darkmode works as well